test/test_clicommand.py

```diff
@@ -58,7 +71,7 @@ def test_scan_facts_default(self):
                                            "--reset", "--reportfile",
                                            "data/test_report.csv", "--facts",
                                            "default", "ansible_forks",
-                                           "100"])
+                                           "100", "--vault", "./vault_pass"])
```

Comment:
Let's use /tmp for these files. I can imagine that writing to `./vault_pass` in some directory we don't control could cause problems for someone later on.

Also, please pull the default file paths for testing out into module-level constants or group the shared code. I know the rest of the code isn't written that way, but I think that it all should be, and at least we can avoid making it worse.